test/regression_tests.sh

```diff
@@ -28,9 +28,6 @@ install_torchserve_from_source() {
   cd serve
   echo "Installing torchserve torch-model-archiver from source"
   ./scripts/install_from_src_ubuntu
-  echo "TS Branch : " $(git rev-parse --abbrev-ref HEAD) >> $3
-  echo "TS Branch Commit Id : " $(git rev-parse HEAD) >> $3
-  echo "Build date : " $(date) >> $3
```

Comment:
What's the reason for reverting this?

Reply:
This got introduced while rebasing from `staging_0_1_1` branch to `master` and didn't show up as a conflict, reverted these unwanted changes.